scripts/ubuntu-22.04-cpp.dockerfile

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The license should probably be changed to the Meta license..

Reply:
Updated